include/Dialects/LinalgTransform/LinalgTransformOps.td

```diff
@@ -33,6 +34,34 @@ class Linalg_Transform_Operation<string name, list<OpTrait> props = []>
 
 //===----------------------------------------------------------------------===//
 
+def ScopeOp : Linalg_Transform_Operation<"scope",
+    [IsolatedFromAbove, OpAsmOpInterface,
+     DeclareOpInterfaceMethods<RegionBranchOpInterface>]> {
+  let description = [{An operation to restrict transformation scopes.}];
+
+  let regions = (region AnyRegion:$body);
+  let arguments = (ins Variadic<AnyType>:$ins);
+  let results = (outs Variadic<AnyType>:$outs);
+  let assemblyFormat = "`(` operands `)` attr-dict-with-keyword $body `:` functional-type(operands, results)";
+
+  let extraClassDeclaration = [{
+    static StringRef getDefaultDialect() { return "linalg_transform"; }
+  }];
```

Comment:
I'm not sure this is relevant since the operation will be wrapping the IR being transformed rather than the IR that describes the transformation.

Reply:
The sole purpose of this is so that the `linalg_transform.forward` required terminator is just `forward`.